File: Python/Easy/SplitTheArray.py
from typing import List


class Solution:
    def isPossibleToSplit(self, nums: List[int]) -> bool:
        freq = {}
        for num in nums:
            freq[num] = freq.get(num, 0) + 1
            if freq[num] > 2:
                return False
        return True

File: JavaScript/Easy/SplitTheArray.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var isPossibleToSplit = function (nums) {
  const freq = {};
  for (const num of nums) {
    freq[num] = (freq[num] || 0) + 1;
    if (freq[num] > 2) {
      return false;
    }
  }
  return true;
};
